src/projetos/HomeProjetosListPage.tsx

```
@@ -320,7 +320,7 @@ export function HomeProjetosListPage() {
                           <PaginationLink
                             isActive={currentPage === page}
                             onClick={() => setCurrentPage(page)}
-                            className="!text-gray-800 hover:!bg-gray-100 rounded-full"
+                             className='border-gray-300 !text-black hover:!bg-gray-100 !rounded-md !px-3 !py-1'
```

ESLint

[error] Replace `·className='border-gray-300·!text-black·hover:!bg-gray-100·!rounded-md·!px-3·!py-1'` with `className="border-gray-300·!text-black·hover:!bg-gray-100·!rounded-md·!px-3·!py-1"`

(prettier/prettier)